package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "unchained-wallets",
-  "version": "0.6.2",
+  "version": "1.0.0",
```

Comment:
Just leave it to release please. Should be fine with a `feat!` prefix.

Reply:
oh yeah. I had forgotten.

Reply:
Changed in 35668a8805582ecc715ddc5e5214ac4de91c22ff